templates/themes/base/tpl/layout.tpl

```diff
@@ -21,11 +21,26 @@
     <meta name="category" content="{meta_category}" />
     <meta name="rating" content="{meta_rating}" />
     <meta name="distribution" content="{meta_distribution}" />
+    <meta name="theme-color" content="#ffffff" />
     {if $header}{header}{/if}
 
     {if $is_pwa_enabled}
       <link rel="manifest" href="{{ $design->url('pwa','main','manifest') }}" />
       {{ $design->staticFiles('js', PH7_LAYOUT . PH7_SYS . PH7_MOD . 'pwa/' . PH7_TPL . PH7_DEFAULT_THEME . PH7_SH . PH7_JS, 'sw-register.js') }}
+       <link rel="apple-touch-icon" sizes="57x57" href="{url_tpl_mod_img}apple-icon-57x57.png">
```

Comment:
For your information, `{url_tpl_mod_img}` variable gives the current module where you currently are. So, when you are on the affiliate module, it will give the img/ folder of the affiliate module (`templates/system/modules/affiliate/themes/base/img/`
), when you are the payment section, it will give the img/ folder of the `payment` module (`templates/system/modules/payment/themes/base/img/`). 


Here, in this situation, we ONLY, and ALWAYS want the `img/` path of the "pwa" module (which is ` templates/system/modules/pwa/themes/base/img/`). Because of that, in order to avoid 404 apple-icon links, you will have to replace the `{url_tpl_mod_img}` to `{{ $design->url('pwa','main','manifest') }}`.


Let me know if there is something not clear :)

For more technical references:
https://github.com/pH7Software/pH7-Social-Dating-CMS/blob/e0dfcc72dcdbe99020a7dce2134501727f9a2513/_protected/framework/Layout/Tpl/Engine/PH7Tpl/Predefined/Variable.class.php#L88
https://github.com/pH7Software/pH7-Social-Dating-CMS/blob/9c3a7d53329c484fecf39931718a1ad31532d51f/_protected/framework/Mvc/Router/FrontController.class.php#L172

Reply:
Oops! Yes, indeed, the path I gave `{{ $design->url('pwa','main','manifest') }}`, is the wrong one.

Should be something like the following:
`{% PH7_LAYOUT . PH7_SYS . PH7_MOD . 'pwa/' . PH7_TPL . PH7_DEFAULT_THEME . PH7_SH . PH7_IMG %}`